docs/configuration/backends/kubernetes.md

```diff
@@ -158,6 +158,7 @@ The following security annotations can be applied to the ingress object to add s
 | `ingress.kubernetes.io/public-key:VALUE`                 | Adds pinned HTST public key header.                                                                                                                                                                 |
 | `ingress.kubernetes.io/referrer-policy:VALUE`            | Adds referrer policy  header.                                                                                                                                                                       |
 | `ingress.kubernetes.io/is-development:false`             | This will cause the `AllowedHosts`, `SSLRedirect`, and `STSSeconds`/`STSIncludeSubdomains` options to be ignored during development.<br>When deploying to production, be sure to set this to false. |
+| `ingress.kubernetes.io/app-root:VALUE`             |  Redirects all requests for / to the defined path. |
```

Comment:
Sorry, I meant to just drop the _on the other hand_ introduction. The remaining parts of the second sentence are good and should persist.